plugin-management-library/src/main/java/io/jenkins/tools/pluginmanager/impl/PluginManager.java

```diff
@@ -271,10 +266,10 @@ public void resolveDependencies(Plugin plugin) {
 
             if (!StringUtils.isEmpty(installedVersion)) {
                 if (comparePluginVersions(installedVersion, dependencyVersion) < 0) {
-                    System.out.println("Installed version of " + dependencyName + " is less than minimum " +
+                    LOGGER.log(Level.INFO, "Installed version of " + dependencyName + " is less than minimum " +
```

Comment:
Better to use formatter APIs so that the strings are not calculated when INFO logging does not happen

```suggestion
                   LOGGER.log(Level.INFO, "Installed version of {0} is less than minimum required version of {1} upgrading bundled dependency", new Object[] {dependencyName , dependencyVersion }) 
```